Program.cs

```diff
@@ -1,7 +1,20 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿using System;
+namespace Training {
+   internal class Program {
+      static void Main (string[] args) {
+         Console.Write ("Input: ");
+         string decimal_number = Console.ReadLine ();
+         if (int.TryParse (decimal_number, out int dec) && dec!=0) {
+            string hexa = dec.ToString ("X");
+            Console.WriteLine ($"HEX: {hexa}");
+            string bin = "";
+            while (dec > 0) {

```

Comment:
Does it support negative values because I see no value for binary if I enter negative value?

Reply:
Changed